c2corg_ui/static/partials/cards/feeds.html

```diff
@@ -0,0 +1,115 @@
+<article class="post" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;type = cardCtrl.getDocumentType(doc.document.type);" ng-click="cardCtrl.openDoc(doc)">
```

Comment:
Many too long lines in this partial, prefer spliting them in a set of shorter lines, it's more readable.